added some of the params undisclosed in the API. still missing more i… (#3380)
opensearch-project/documentation-website: _observing-your-data/alerting/api.md

Returns an array of all alerts.

> **Query Parameters**

Reviewer: Might be a good idea to have some examples of how these path parameters are used in the request. See [Path and HTTP methods](https://github.com/opensearch-project/documentation-website/blob/main/API_STYLE_GUIDE.md#path-and-http-methods) in API_STYLE_GUIDE.md.

Parameter | Data type | Description
:--- | :--- | :---

> sortOrder     | string | Determines the order of the results. can be "asc" or "desc". defaults to "asc"

Reviewer: Global comment:
In parameter and request/response field tables, let's use the back ticks to set off parameter and field names in code style: `sortOrder`
For an example, see the parameters in [Query string queries](https://opensearch.org/docs/latest/query-dsl/full-text/query-string/).

Suggestion:
If you add one space before and one space after the parameter name, they'll render just find on the website; the column widens to the longest entry and proper spaces are added after the shorter entries.

| `sortOrder` | String | Determines the order of the results. Options are `asc` or `desc`. Defaults to `asc`. |

missing       | string | Optional
size          | string | Determines the size of the request to be returned. defaults to 20
startIndex    | string | The start index to start from. used for paginating results. defaults to 0

> searchString  | string | Search string for looking for a specific alert. defaults to empty string

Reviewer: ```suggestion
searchString  | String | A search string used to look for a specific alert. Defaults to an empty string.
```
Gobal comment:
For the Data Type column, use initial caps: e.g., String, Array, Boolean, Object, etc.

severityLevel | string | Severity level to filter for. defaults to "ALL"
alertState    | string | Alert state to filter for. defaults to "ALL"

> monitorId     | string | Filter by monitorId

Reviewer 2: ```suggestion
monitorId     | string | Filter by monitor ID.
```

Returns an array of all alerts.

**Path parameters**
The following table lists the available path parameters. All path parameters are optional.
Parameter | Data type | Description
:--- | :--- | :---
`sortString` | String | Determines how to sort the results. Defaults to `monitor_name.keyword`.
`sortOrder` | String | Determines the order of the results. Options are `asc` or `desc`. Defaults to `asc`.

> `missing` | String | Optional

Reviewer 3: ```suggestion
`missing` | String | Optional.
```

`size` | String | Determines the size of the request to be returned. Defaults to `20`.

> `startIndex` | String | The start index to start from. Used for paginating results. Defaults to `0`.

Reviewer 3: Can we remove the first instance of "start"?

`searchString` | String | A search string used to look for a specific alert. Defaults to an empty string.

> `severityLevel` | String | Severity level to filter for. Defaults to `ALL`.

Reviewer 3: ```suggestion
`severityLevel` | String | The severity level to filter for. Defaults to `ALL`.
```

> `alertState` | String | Alert state to filter for. Defaults to `ALL`.

Reviewer 3: ```suggestion
`alertState` | String | The alert state to filter for. Defaults to `ALL`.
```

> `monitorId` | String | Filter by monitor ID.

Reviewer 3: ```suggestion
`monitorId` | String | Filters by monitor ID.
```
